decimal.go

```diff
@@ -1307,6 +1307,31 @@ func (d NullDecimal) MarshalJSON() ([]byte, error) {
 	return d.Decimal.MarshalJSON()
 }
 
+// UnmarshalText implements the encoding.TextUnmarshaler interface for XML
+// deserialization
+func (d *NullDecimal) UnmarshalText(text []byte) error {
+	str := string(text)
+
+	if str == "" {
```

Comment:
So this is basically the case for empty XMLs and XMLs withotu the body e.g `<test></test>`?
Could you add a short comment that would denote that, so it will be easier to refactor in the future if needed? :D

Reply:
Done.